models/app.model.js

```diff
@@ -30,10 +30,14 @@ exports.selectArticles = () => {
 exports.selectArticleById = (articleId) => {
 
     const sqlString = `
-        SELECT * FROM articles
-        where articles.article_id = $1;
-
-    `
+        SELECT articles.*, COUNT(comments.article_id) as comment_count 
```

Comment:
Since the "as" here is also SQL, it should be capitalised consistently with the rest of the SQL statement please